stdlib/ballerina-http/src/main/ballerina/http/http_request.bal

```diff
@@ -156,32 +156,33 @@ public type Request object {
     public function getContentType() returns string;
 
     documentation {
-        Gets the request payload as a `json`.
+        Extracts `json` payload from the request. If the payload is not json compatible, an `error` is returned.
```

Comment:
What does it mean to say JSON compatible? Shouldn't it simply be "If the payload is not a JSON, an `error` is returned"? And same in other similar places?